tests/functional_tests/missing_files.bats

```diff
@@ -33,8 +48,16 @@ teardown() {
 
 #Note : The output of this test will differ if any changes are made to docs/examples/unit_tests/execute.sh
 @test "run no test info found" {
-  sed -i '/bash execute.sh "$language"/ i rm -f ./test_info.txt' ../../execution_nodes/extract_run.sh
-  sed -i '/bash execute.sh "$language"/ a cp ./results/log.txt /tmp/missing-files/no-test-info-found-log.txt' ../../execution_nodes/extract_run.sh
+  sed -i '/bash execute.sh "$language"/ i rm -f ./test_info.txt' ../../execution_nodes/execution_node_1/extract_run.sh
```

Comment:
There seems to be too much of repetition here? would a shell function help?

Reply:
Will it work if we keep only one copy for extract_run.sh for all the execution nodes in the parent directory execution_nodes? We reduce the duplication of that script, and to make matters simple, we can add another parameter to the extract_run.sh script, which specifies from which node the request is coming from, so the evaluation can happen inside the specific execution node directory.